src/js/apps/admin/program/workflows/workflows_app.js

```diff
@@ -17,12 +17,16 @@ export default App.extend({
     this.getRegion('content').startPreloader();
   },
   beforeStart({ program }) {
-    return Radio.request('entities', 'fetch:programActions:collection:byProgram', { programId: program.id });
+    return [
+      Radio.request('entities', 'fetch:programActions:collection:byProgram', { programId: program.id }),
+      Radio.request('entities', 'fetch:programFlows:collection', { program: program.id }),
+    ];
   },
-  onStart({ program }, actions) {
+  onStart({ program }, [actions], [flows]) {
     this.actions = actions;
 
-    this.showChildView('content', new ListView({ collection: actions }));
+    const collection = new Backbone.Collection([...actions, ...flows]);
```

Comment:
I think both of these should be `...actions.models` and `..flows.models`  But does this work?  I can't figure out why..

Reply:
Huh, I'm not sure why it works either. I'll change it to models.